docs/man/borg-init.1

```diff
@@ -43,17 +43,12 @@ Encryption can be enabled at repository init time. It cannot be changed later.
 It is not recommended to work without encryption. Repository encryption protects
 you e.g. against the case that an attacker has access to your backup repository.
 .sp
-But be careful with the key / the passphrase:
-.sp
-If you want "passphrase\-only" security, use one of the repokey modes. The
-key will be stored inside the repository (in its "config" file). In above
-mentioned attack scenario, the attacker will have the key (but not the
-passphrase).
-.sp
-If you want "passphrase and having\-the\-key" security, use one of the keyfile
-modes. The key will be stored in your home directory (in .config/borg/keys).
-In the attack scenario, the attacker who has just access to your repo won\(aqt
-have the key (and also not the passphrase).
+The repository is encrypted with a key, and the key is protected with a passphrase. 
+By setting the encryption mode, you can choose whether to store the key inside the
+repository ("passphrase only" or "repokey" mode) or to store the key separately in 
```

Comment:
this will confuse people about the mode name.

Reply:
Maybe you're right. I re-used the terminology from the existing manpage. Perhaps it would be better to just say "repokey" mode. i.e. don't mention "passphrase only" at all.